util/mmio.h

```diff
@@ -90,38 +144,56 @@ static inline void s390_mmio_read(const void *mmio_addr, void *val,
 #define MAKE_WRITE(_NAME_, _SZ_)                                               \
 	static inline void _NAME_##_be(void *addr, __be##_SZ_ value)           \
 	{                                                                      \
-		s390_mmio_write(addr, &value, sizeof(value));                  \
+		if (likely(s390_mio_supported))                                \
```

Comment:
Do not annotate with likely, you can't know at compile time which leg the running system will prefer but the branch predictor can..

Reply:
Makes sense, will do.